apps/demos/Demos/Splitter/Overview/React/PaneContent.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useMemo } from 'react';
+
+export default function PaneContent(paneName: string) {
+  const dimensionOptions = new Set(['size', 'minSize', 'maxSize'])
+
+  return function template(data) {
+    const getPaneState = useMemo(() => {  
+      if (data.resizable !== false && !data.collapsible) {
+        return 'Resizable only';
+      }
+      return `${data.resizable ? 'Resizable' : 'Non-resizable'} and ${data.collapsible ? 'collapsible' : 'non-collapsible'}`;
```

Comment:
Could u please simplify such calculations everywhere? It's really difficult to read at the moment.
Maybe like this?
```suggestion
      const resizableText = data.resizable ? 'Resizable' : 'Non-resizable';
      const collapsibleText = data.collapsible ? 'collapsible' : 'non-collapsible';
      
      return `${resizableText} and ${collapsibleText}`;
```